dist/resources/orders.js

```diff
@@ -70,10 +70,10 @@ module.exports = function (api) {
       var amount = params.amount,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Again, changes in dist will be automatic

Reply:
@sidag95 i have removed thoses changes